tools/rapids-download-wheels-from-github

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# A utility script that downloads wheel artifacts from Github Artifacts to a temporary directory and returns the path to the directory
+set -eo pipefail
+
+source rapids-prompt-local-repo-config
+
+pkg_type="$1"
+if [ "${pkg_type}" = "cpp" ] || [ "${pkg_type}" = "python" ]; then
+    # remove pkg_type from args because we handle it in this script
+    shift;
+    pkg_name="$(rapids-package-name "wheel_${pkg_type}")"
+fi
```

Comment:
I think this logic can be cleaned up a bit more. Now that we've removed the legacy components, can you change this to a `case` statement to validate that the first positional argument is either `cpp` or `python`? Anything else should print an error message and exit with a non-zero code.